examples/cityscapes/leap_binder.py

```diff
@@ -0,0 +1,308 @@
+from typing import List, Dict, Callable
+from PIL import Image
+import numpy as np
+import json
+import tensorflow as tf
+from tensorflow import Tensor
+
+from utils_all.gcs_utils import _download
+from utils_all.metrics import regression_metric, classification_metric, object_metric, od_loss, calculate_iou, \
+    convert_to_xyxy
+from utils_all.preprocessing import Cityscapes, load_cityscapes_data, CATEGORIES, CATEGORIES_no_background, \
+    CATEGORIES_id_no_background
+from project_config import IMAGE_STD, IMAGE_MEAN, IMAGE_SIZE, BACKGROUND_LABEL, SMALL_BBS_TH
+from utils_all.general_utils import extract_bounding_boxes_from_instance_segmentation_polygons, \
+    bb_array_to_object, get_predict_bbox_list
+from visualizers.visualizers import bb_decoder, gt_bb_decoder, bb_car_gt_decoder, bb_car_decoder
+
+from code_loader import leap_binder
+from code_loader.contract.datasetclasses import PreprocessResponse
+from code_loader.contract.enums import (
+    DatasetMetadataType,
+    LeapDataType
+)
+
+# ----------------------------------------------------data processing--------------------------------------------------
+def load_cityscapes_data_leap() -> List[PreprocessResponse]:
+    all_images, all_gt_images, all_gt_labels, all_gt_labels_for_bbx, all_file_names, all_metadata, all_cities =\
+        load_cityscapes_data()
+
+    # train_len = len(all_images[0])
+    # val_len = len(all_images[1])
+    # test_len = len(all_images[2])
+
+    train_len = 700
+    val_len = 100
+    test_len = 200
+
+    responses = [PreprocessResponse(length=train_len, data={
```

Comment:
Since this is a big long maybe use list comprehension?
```
lengths = [train_len, val_len, test_len]
[PreprocessResponse(length=[lengths[i]], data={"KEY": VALUE_LIST[i]} for i in range(3))
```

Reply:
done